Workbooks/Virtual Machines - Network Dependencies/Connections Overview - Single VM/Connections Overview.workbook

```diff
@@ -74,12 +75,26 @@
             ],
             "isHiddenWhenLocked": true,
             "queryType": 0,
-            "resourceType": "microsoft.resources/subscriptions"
+            "resourceType": "microsoft.resources/subscriptions",
+            "value": null
+          },
+          {
+            "id": "b12f03d2-87cb-4a88-b4c1-6e24b28cb831",
+            "version": "KqlParameterItem/1.0",
+            "name": "ConnectionLogs",
+            "type": 1,
+            "query": "VMConnection\r\n| where _ResourceId contains \"{WorkbookResource}\"\r\n| take 10\r\n| count",
+            "crossComponentResources": [
```

Comment:
these all have hardcoded resources in the json.  these should be using the subs parameter or some other resource parameter to function correctly